public/static/docs/user-guide/external-dependencies.md

```diff
@@ -151,8 +151,8 @@ determine whether the source has changed and we need to download the file again.
 ## Example: Using import
 
 `dvc import` can download a <abbr>data artifact</abbr> from any <abbr>DVC
-repository</abbr>. It also creates an external dependency in its <abbr>import
-stage</abbr> (DVC-file).
+Repository</abbr> or <abbr>Git repository</abbr>. It also creates an external
+dependency in its <abbr>import stage</abbr> (DVC-file).
```

Comment:
No need to capitalize "repository" here. Actually, please use

`...from any <abbr>DVC project</abbr> or Git repository. It also...`

Thanks

Reply:
Pending. Resolving here, will reopen on current version.